components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/AuthzUtil.java

```diff
@@ -455,6 +455,9 @@ private static boolean isDoGetGroupListOfUserNotImplemented(UserStoreException e
      */
     public static boolean isLegacyAuthzRuntime() {
 
+        if (CarbonConstants.ENABLE_LEGACY_AUTHZ_RUNTIME == null) {
```

Comment:
Can this be null in the runtime?

Reply:
Fixed by - https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2430/commits